src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,25 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard/MovieCard';
+import { MovieCardShape } from '../../shapes/MovieCardShape';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <ul className="movies">
+    {movies.map(movie => (
+      <li className="card" key={movie.imdbId}>
+        <MovieCard {...movie} />
+      </li>
+    ))}
+  </ul>
 );
+
+MoviesList.propTypes = {
+  movies: PropTypes.arrayOf(
+    PropTypes.shape = MovieCardShape,
+  ),
```

Comment:
это работает? лучше так не делать
```suggestion
  movies: PropTypes.arrayOf(MovieCardShape),
```